src/debug/getters.yaml

```diff
@@ -55,3 +89,83 @@
       type: array
       items:
         $ref: '#/components/schemas/BadBlock'
+  examples:
+    - name: debug_getBadBlocks example
+      params: []
+      result:
+        name: Bad block array
+        value:
+          - block:
```

Comment:
Bad block consists of three fields according to the schema:
https://github.com/ethereum/execution-apis/blob/7a68a3ea7f638ad975a26729aa5a3902d7da8642/src/schemas/block.yaml#L140-L148

The block content in the example doesn't match the schema. I don't know where the issue is coming from, i.e. whether it is with Besu or with the spec. cc @lightclient What would be the response to `debug_getBadBlocks` in geth? I tried to find it with no luck

Reply:
The fix https://github.com/ethereum/execution-apis/pull/469

Reply:
resolved! thanks